fedimint-client/src/meta.rs

```diff
@@ -227,6 +227,7 @@ impl<S: MetaSource + ?Sized> MetaService<S> {
 #[non_exhaustive]
 pub struct LegacyMetaSource {
     reqwest: reqwest::Client,
+    extra_override_keys: Vec<String>,
```

Comment:
Unrelated, slipped in, I assume? @elsirion

Reply:
Oh yeah :see_no_evil: